app/scripts/contentscript.js

```diff
@@ -73,6 +73,6 @@ function isAllowedSuffix (testCase) {
   if (doctype) {
     return doctype.name === 'html'
   } else {
-    return false
+    return true
```

Comment:
This line would attempt to inject in any file without a doctyoe. What types of files are those? Surely HTML is already covered with line 74?